Update MultiInstanceConnecting (#398)
tarantool/cartridge-java: docs/MultiInstanceConnecting.md

# Connecting to multiple Tarantool instances

This section shows examples of how to connect to multiple [Cartridge](https://github.com/tarantool/cartridge) routers.  
We recommend to use load balancer in production applications rather than leaving this task to the connector.

> In case of using **multiple instances of TarantoolClient** we strongly recommend to **shuffle** list of addresses before providing them to clients.

Reviewer: ```suggestion
In the case of using **multiple instances** of `TarantoolClient` we strongly recommend to apply a **shuffle** function to the list of addresses.
```

> By default TarantoolClient uses [round-robin](https://en.wikipedia.org/wiki/Round-robin_scheduling) approache to separate load between Tarantool instances.

Reviewer: ```suggestion
By default, `TarantoolClient` uses the [round-robin](https://en.wikipedia.org/wiki/Round-robin_scheduling) approach for selecting the next available connection to distribute the load between Tarantool instances.
```

> Without addresses **shuffle** the first request of each of your clients will be sent to the same instance.

Reviewer: ```suggestion
When several instances of `TarantoolClient` are used simultaneously, with the default connection selection strategy and without **shuffling** of the server addresses the first request from every instance will be sent to the same server.
```

Reviewer: What problem do the simultaneous requests cause? We need to add a line about that, currently the readers will be confused

> You do not have to worry about shuffling addresses if you are using single TarantoolClient connected to multiple instances.

Reviewer: ```suggestion
You do not have to worry about shuffling addresses if you are using a single `TarantoolClient` connected to multiple instances, or if you are using a custom connection selection strategy that takes into account the problem of simultaneous requests.
```

This section shows examples of how to connect to multiple [Cartridge](https://github.com/tarantool/cartridge) routers.  
We recommend to use balancer in production applications rather than leaving this task to the connector.

If you are using **multiple tarantool clients** do not forget to **shuffle addresses** before providing them to clients.

> TarantoolClient uses [round-robin](https://en.wikipedia.org/wiki/Round-robin_scheduling) so without URIs shuffle

Reviewer 2: ```suggestion
By default TarantoolClient uses [round-robin](https://en.wikipedia.org/wiki/Round-robin_scheduling) approach to separate load between Tarantool instances.

```

> the first request of each of your clients will be sent to the same instance.

Reviewer 3: ```suggestion
Without addresses **shuffle** the first request of each of your clients will be sent to the same instance.

Reviewer 2: ```suggestion
> **Warning**
> Without addresses **shuffle** you may have a situation when clients will send something to the same instances. 
> Especially when you start your clients in parallel at the same time.
```
